lib/Applify.pm

```diff
@@ -321,12 +321,14 @@ sub _generate_application_class {
 
     for my $option (@{$self->{options}}) {
       my $name = $option->{name};
-      my $fqn = join '::', $application_class, $name;
       if ($meta) {
         $meta->add_attribute($name => {is => 'rw', default => $option->{default}});
       }
       else {
+        my $fqn = join '::', $application_class, $name;
         _sub($fqn => sub { @_ == 2 and $_[0]->{$name} = $_[1]; $_[0]->{$name} });
+        $fqn = join '::', $application_class, join '_', has => $name;
+        _sub($fqn => sub { !!(exists $_[0]->{$name} && defined $_[0]->{$name}) });
```

Comment:
Why do you need both exists and defined? I'm guessing one of them should be enough, though I cannot decide on which is the best option.

Reply:
`defined` as every one gets a *default*
https://github.com/jhthorsen/applify/blob/df2b22ba4ea09a34e4745332ebbe4775496887b4/lib/Applify.pm#L130